ServiceProviderBot/Bot/Dialogs/UpdateOrganization/UpdateOrganizationDialog.cs

```diff
@@ -39,6 +39,16 @@ public override WaterfallDialog GetWaterfallDialog()
                     return await BeginDialogAsync(stepContext, UpdateCapacityDialog.Name, null, cancellationToken);
                 },
                 async (stepContext, cancellationToken) =>
+                {
+                    var organization = await database.GetOrganization(stepContext.Context);
+                    if(organization.HasJobTrainingServices)
```

Comment:
James added a NeedsUpdate method in his change that checks if the individual services need to update. If you pull the latest from develop into your branch (git pull --rebase origin develop) then you can add you check into that